### ISSUE TITLE
dotnet dev-certs started failing in macOS

### ISSUE THREAD

**Issue by kblok** [actions/virtual-environments#1690]
**Description**  
The `dotnet dev-certs` started to fail here https://github.com/hardkoded/playwright-sharp/pull/878/checks?check_run_id=1176770064 from one day to the other.

This is part of the workflow definition

```
jobs:
  chromium-mac:
    name: Chromium on MacOS
    runs-on: macOS-latest
    steps:
      - uses: actions/checkout@v2
      - name: Setup .NET Core
        uses: actions/setup-dotnet@v1
        with:
          dotnet-version: 3.1.100
      - name: Create Certificate
        run: |
          dotnet dev-certs https --clean
          dotnet dev-certs https -ep src/PlaywrightSharp.TestServer/testCert.cer
````

You can see the rest here https://github.com/hardkoded/playwright-sharp/blob/master/.github/workflows/tests.yml

**Area for Triage**:  
.NET Core

**Virtual environments affected**  
- [X] macOS 10.15
- [ ] Ubuntu 16.04 LTS
- [ ] Ubuntu 18.04 LTS
- [ ] Ubuntu 20.04 LTS
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Comment by maxim-lobanov** [comment]
Potentially, can be related to https://github.com/actions/virtual-environments/pull/1630

**Comment by maxim-lobanov** [comment]
@kblok , Checked and confirmed that issue is not related to PR above, your builds run on image 20200918.1 but PR was merged 21 September.

For now we don't have an idea about root cause of this issue but we are sure that this issue is not caused by image update:
1) Green build on image version `20200918.1`: https://github.com/hardkoded/playwright-sharp/actions/runs/269627718
2) Red build on image version `20200918.1`: https://github.com/hardkoded/playwright-sharp/runs/1176888266?check_suite_focus=true

The same image version means that builds are run on the identical VM.

**Comment by kblok** [comment]
Thanks for the feedback @maxim-lobanov.

**Comment by kblok** [comment]
The output of `actions/setup-dotnet@v1` looks different. Do you think I should create this issue on https://github.com/actions/setup-dotnet ?

**Comment by maxim-lobanov** [comment]
@kblok , yes, I have confirmed that `actions/setup-dotnet` is root cause of this issue.

Please try to specify `uses: actions/setup-dotnet@v1.6.0` directly in your workflow and build will pass.
I have confirmed it in my fork: https://github.com/maxim-lobanov/playwright-sharp/runs/1177397219?check_suite_focus=true

Looks like actions/setup-dotnet@v1.7.0 was released recently and `v1` tag was updated to follow it.
cc: @zeisinger as owner of actions/setup-dotnet. Looks like a breaking changes in last version. Could you please take a look?

**Comment by ZEisinger** [comment]
Please open an issue. v1.7.1 is the latest version and fixes known problems from v1.7.0
